.eslintignore

```diff
@@ -50,5 +50,6 @@ ReactNativeClient/pluginAssets/
 
 # Ignore files generated from TypeScript files
 ElectronClient/app/gui/ShareNoteDialog.js
+ElectronClient/app/gui/NotebookPropertiesDialog.js
```

Comment:
Please rename to FolderPropertiesDialog. There should be no mention of "notebook" anywhere in the code itself.